packages/langium-cli/src/generator/module-generator.ts

```diff
@@ -21,10 +21,16 @@ export function generateModule(grammar: langium.Grammar, config: LangiumConfig):
     node.append(
         'import { ', grammar.name, "AstReflection } from './ast';", NL,
         'import { ', grammar.name, "GrammarAccess } from './grammar-access';", NL,
-        'import { ', grammar.name, "LanguageMetaData } from './meta-data';", NL,
         "import { Parser } from './parser';", NL, NL
     );
 
+    node.append('const metaData = {', NL);
```

Comment:
Shall we export this so you can access it directly in your language implementation?